addons/mail/static/src/components/discuss/discuss.js

```diff
@@ -26,7 +26,7 @@ export class Discuss extends Component {
         this.discuss.update({ isOpen: true });
         if (this.discuss.thread) {
             this.trigger('o-push-state-action-manager');
-        } else if (!this._activeThreadCache && this.discuss.messaging.isInitialized) {
+        } else if (!this._activeThreadCache && this.discuss.messaging.isInitialized && this.discuss.activeMobileNavbarTabId === 'mailbox') {
```

Comment:
The condition on active tab seems a little bit too weak, we probably want a new field on discuss to determine whether the init thread was already opened once, and then not do it again?

Reply:
@seb-odoo changes done!!